src/importexport/guitarpro/internal/guitarbendimport/benddatacollector.cpp

```diff
@@ -196,7 +196,6 @@ void BendDataCollector::regroupBendDataByTiedChords()
                     }
                 }
 
-                ChordImportedBendData chordBendData;
                 chordBendData.chord = currentNote->chord();
```

Comment:
`chordBendData` is here an object that has been moved from (see line 173), which means that it is in an unspecified state. Reusing it, may therefore not be a good idea. Instead, you could limit the scope of the outer `chordBendData, by putting a `{` before line 166 and a `}` after the current line 173 (`chunk.push_back(std::move(chordBendData));`).

Reply:
Isee, will change

Reply:
done